pkg/koordlet/util/node.go

```diff
@@ -58,6 +66,23 @@ func GetRootCgroupCurCPUSet(qosClass corev1.PodQOSClass) ([]int32, error) {
 	return cpuset.ParseCPUSet(cpus), nil
 }
 
+// GetBeCgroupCurCPUSet gets the current cpuset of besteffort podQoS' cgroup.
+func GetBeCgroupCurCPUSet() ([]int32, error) {
```

Comment:
How about `GetBECgroupCurCPUSet`? It would be appreciated if we could unify the naming of the `BE`.

Reply:
done